### PR TITLE
Baby/Parent API for Ageable entities.

### PR DESCRIPTION
**Pull request by aramperes**
See GlowstonePlusPlus PR [here](https://github.com/GlowstoneMC/GlowstonePlusPlus/pull/231).

Will be used for Baby AI in the future.

### CODE REVIEW

**Comment by mastercoms** [comment]
Why is there not a setBaby method?

**Comment by aramperes** [comment]
? https://github.com/GlowstonePlusPlus/Glowkit/blob/1d8635c2594d9d544cc5eaa2f3d58a84b1ff2268/src/main/java/org/bukkit/entity/Ageable.java#L39

**Comment by mastercoms** [comment]
Ah, I was just looking at the diff :blush:

File: src/main/java/org/bukkit/entity/Ageable.java [hunk 1]
```diff
@@ -64,4 +64,22 @@
      * @param breed breedability of the animal
      */
     public void setBreed(boolean breed);
+
+    /**
+     * Creates a baby with the same type and location as the parent.
```
Comment (mastercoms) [inline src/main/java/org/bukkit/entity/Ageable.java]: Label this as a convenience method.

**Comment by mastercoms** [comment]
Nice job! Thank you for your contribution.

**Pull request merged**